sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -843,11 +843,10 @@ class HiveDDLSuite
       // The table type is not changed to external
       assert(
         catalog.getTableMetadata(TableIdentifier(tabName)).tableType == CatalogTableType.MANAGED)
-      // The table property is case sensitive. Thus, external is allowed
-      sql(s"ALTER TABLE $tabName SET TBLPROPERTIES ('external' = 'TRUE')")
-      // The table type is not changed to external
-      assert(
-        catalog.getTableMetadata(TableIdentifier(tabName)).tableType == CatalogTableType.MANAGED)
+      val e1 = intercept[ParseException] {
+        sql(s"ALTER TABLE $tabName SET TBLPROPERTIES ('external'='bar')")
```

Comment:
I think `'External'='bar'` still works?

Reply:
yeah. 'External'='bar' still works. `EXTERNAL` is the key word of the hive metastore.

Reply:
then let's use `External` and keep this test.

Reply:
ok.